hive/envs/gym_env.py

```diff
@@ -21,6 +21,7 @@ def __init__(self, env_name, num_players=1, **kwargs):
         """
         self.create_env(env_name, **kwargs)
         super().__init__(self.create_env_spec(env_name, **kwargs), num_players)
+        self._seed = seeder.get_new_seed("environment")
```

Comment:
This line shouldn't be here. This file should not be calling the seeder. The seed should be stored in the seed() method

Reply:
resolved.